### ISSUE TITLE
"I'm a experienced Security practicioner and want to learn to code, where do I start"

### ISSUE THREAD

**Issue by DinisCruz** [DinisCruz/Book_SecDevOps_Risk_Workflow#13]
> James just asked me this

Add more detail on where to start

**Comment by davevs** [comment]
These kind of questions (and other flavours) are posted regularly on Reddit and Stackoverflow. The answers vary from use source X to just do it. In my opinion you should take your curiosity take you on the road that suits you best. 
For example; I'm training myself python by converting a ruby based cheat framework for a game I play into python. :)

**Comment by davevs** [comment]
If you need to give at least 1 source I'd go with tutorialspoint; they cover a lot of languages with an online testing sandbox.

**Comment by DinisCruz** [comment]
yeah, but we need to create a better road-map on how to become an appsec professional.

One path that I think is really good is to start by being a Security Champion

**Comment by DinisCruz** [comment]
in https://twitter.com/colindomoney/status/792830598729596928

> Interesting. How do you start a ‘non-programmer’ in grokking programming? CBT, Coursera, a project to hack ?

Depends, if you want to go into a security path, I would say start to exploit vuln-by-design apps and build websites
